cmd/registry/cmd/compute/score.go

```diff
@@ -0,0 +1,75 @@
+package compute
+
+import (
+	"context"
+
+	"github.com/apigee/registry/cmd/registry/core"
+	"github.com/apigee/registry/cmd/registry/patterns"
+	"github.com/apigee/registry/cmd/registry/scoring"
+	"github.com/apigee/registry/connection"
+	"github.com/apigee/registry/log"
+	"github.com/apigee/registry/rpc"
+	"github.com/spf13/cobra"
+)
+
+func scoreCommand() *cobra.Command {
+	return &cobra.Command{
+		Use:   "score (experimental)",
```

Comment:
I don't think we should use "(experimental)" here since this is building out an agreed-upon design.

Reply:
I was planning to take it out once the tool was ready. Basically something to represent that this is WIP. I had seen the use of term `experimental` before and thought of following the same. I can change it to something else.

Reply:
Updated this to `(wip)` instead.